src/floating-focus.js

```diff
@@ -188,8 +188,7 @@ export default class FloatingFocus {
 
 	getFloaterPosition(target) {
 		const targetStyle = window.getComputedStyle(target);
-		// default 4 px padding
-		const padding = parseFloat(targetStyle.outlineOffset) || 4;
+		const padding = parseFloat(targetStyle.outlineOffset || '0px');
```

Comment:
Ik bedenk me net: Is de default niet 0?

Reply:
Yeah, omdat het computed style is zou die '0px' fallback in principe nooit moeten triggeren -- alleen bijv in de tests komt er wel echt `null` uit, dus het lijkt me een prima guard om erin te houden